### PR TITLE
Convert to Pest

### PR DESCRIPTION
**Pull request by alexmanase**
It think it will be better to have tests using Pest, because I think it will be easier to maintain. Also, the code seems cleaner.

### CODE REVIEW

**Comment by alexmanase** [comment]
@driesvints don't you think it will be better to use snapshot testing for components(https://pestphp.com/docs/snapshot-testing)?

If you think it's a good idea, then I add this to the PR.

Thank you!

File: src/Components/Editors/EasyMDE.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-declare(strict_types=1);
```
Comment (driesvints) [inline src/Components/Editors/EasyMDE.php:3]: Why were all of these removed? Can you please add them back?

File: tests/Components/Alerts/AlertTest.php [hunk 2]
```diff
@@ -2,84 +2,65 @@
...
-        $this->assertComponentRenders($expected, '<x-alert/>');
-    }
-
-    #[Test]
-    public function we_can_specify_a_type()
-    {
-        session()->flash('error', 'Form contains some errors.');
+    assertComponentRenders($expected, '<x-alert/>');
+});
+test('we can specify a type', function () {
```
Comment (driesvints) [inline tests/Components/Alerts/AlertTest.php:16]: Please leave one empty line between all tests.

File: tests/Pest.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+use Gajus\Dindent\Indenter;
+use Tests\Components\ComponentTestCase;
+use Tests\TestCase;
+
+/*
+|--------------------------------------------------------------------------
+| Test Case
+|--------------------------------------------------------------------------
+*/
```
Comment (driesvints) [inline tests/Pest.php:11]: Remove all of the comments in this file.

**Review by driesvints — CHANGES_REQUESTED** [review]

File: src/Components/Editors/EasyMDE.php (hunk 1 above)
Comment (alexmanase) [reply src/Components/Editors/EasyMDE.php]: Sorry, by mistake.

File: tests/Components/Alerts/AlertTest.php (hunk 2 above)
Comment (alexmanase) [reply tests/Components/Alerts/AlertTest.php:17]: The Drift plugin makes the tests like that. I reviewed all the tests and added spaces.

File: tests/TestView.php [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-declare(strict_types=1);
+
```
Comment (driesvints) [inline tests/TestView.php:3]:
```suggestion
 declare(strict_types=1);
```

**Comment by driesvints** [comment]
Amazing work. Thank you @alexmanase

**Comment by alexmanase** [comment]
Happy to assist!

**Pull request merged**